bin/help.ml

```diff
@@ -78,17 +78,15 @@ let release =
         \        line.";
       `P
         "If everything went well you can now publish the distribution and\n\
-        \        its documentation on the WWW.";
+        \        its documentation to Github.";
       `Pre "dune-release publish";
       `P
-        "The distribution is now public. It may already have been picked up\n\
-        \        by other systems hence do not try to alter the archive and\n\
-        \        republish it with a different bit-stream after that point (if\n\
-        \        you are tempted to do this please consider taking a functional\n\
-        \        programming course). At worst 410 the archive from\n\
-        \        the WWW. But in most cases, if there is a problem with the\n\
-        \        archive, simply leave it there and publish a new one with an\n\
-        \        increased patch version number.";
+        "The distribution is now public. It may already have been picked up by \
+         other systems hence do not try to alter the archive and republish it \
+         with a different bit-stream after that point. At worst 410 the \
```

Comment:
Since this is being touched: I would replace "bit-stream" with "content" because that's a more common term and is more understandable.